frontend/packages/local-storage-operator-plugin/src/components/disks-list/disks-list-page.tsx

```diff
@@ -131,17 +156,21 @@ export const NodesDisksListPage: React.FC<NodesDisksListPageProps> = ({
         <>
           <p>{t('lso-plugin~Disks Not Found')}</p>
           {csvName && operatorNs && nodeRole !== 'master' && (
-            <Link
-              className="co-m-primary-action"
-              to={`/k8s/ns/${operatorNs}/clusterserviceversions/${csvName}/${referenceForModel(
-                LocalVolumeDiscovery,
-              )}/~new`}
+            <Button
+              isDisabled={isDiscoveryDisabled}
```

Comment:
You can just do this:
```suggestion
              isDisabled={inProgress}
```